src/index.js

```diff
@@ -1,12 +1,33 @@
 const ethers = require("ethers");
 const airnodeAbi = require("@api3/airnode-abi");
+const { CHAINS } = require("@api3/chains");
 const { deriveWalletAddressFromSponsorAddress } = require("./airnode");
 const { nodaryAirnodeAddress, nodaryXPub } = require("../data/metadata.json");
 const nodaryEndpoints = require("../data/endpoints.json");
 const nodaryFeeds = require("../data/feeds.json");
+const nodaryChainAliases = require("../data/chains.json");
 
 const ONE_DAY_IN_SECONDS = 24 * 60 * 60;
 
+function nodaryChainIds() {
+  let mainnetChainIds = [];
+  let testnetChainIds = [];
+
+  return nodaryChainAliases.map((nodaryChainAlias) => {
+    const chain = CHAINS.find((chain) => chain.alias === nodaryChainAlias);
+
+    if (chain.testnet) {
+      testnetChainIds.push(chain.id);
+    } else {
+      mainnetChainIds.push(chain.id);
+    }
+    return {
+      mainnet: mainnetChainIds.sort(),
+      testnet: testnetChainIds.sort(),
+    };
```

Comment:
Just return the chain IDs, don't group them as mainnets/testnets. The user can refer to `@api3/chains` for that.

Reply:
changed the code to return the following response but cannot push it as this repo has been moved to `@api3/dao`;
`[
  '1',        '10',    '30',
  '31',       '56',    '97',
  '100',      '137',   '250',
  '420',      '1101',  '1284',
  '1285',     '1287',  '1442',
  '2221',     '2222',  '4002',
  '5000',     '5001',  '8453',
  '10200',    '42161', '43113',
  '43114',    '59140', '59144',
  '80001',    '84531', '421613',
  '11155111'
]`